### ISSUE TITLE
EL8 RPM packages missing from latest release

### ISSUE THREAD

**Issue by grollinger** [tonarino/innernet#197]
I'm running innernet on a few machines on CentOS and Rocky Linux so I very much appreciated that past releases provided RPM packages for those systems (el8 RPMs).

However, these packages are missing from the latest release [v1.5.3](https://github.com/tonarino/innernet/releases/tag/v1.5.3).

Was that simply an oversight, or are you dropping support for those?
If so, what might be the reason for that?

Thanks for this awesome project, in any case.

**Comment by mcginty** [comment]
Hey @grollinger! Thanks, I'm glad you're finding the project useful.

The RPM builds broke in the last release, so I just disabled them for the time being since I'm not a CentOS/Rocky/Fedora/Redhat user myself and only have so much capacity to support the various package managers on my own. The previous RPM builds especially had multiple different versions for different distros and I ended up having no idea which ones were actually used by anyone.

If you are interested in hosting a RPM/yum repository for innernet, I'm happy to collaborate on that. I'd also like to make `cargo install innernet` more complete (having code in the binary to generate systemd units, etc).

Let me know if you have any recommendations!
